### PR TITLE
Up unit test setup

### PR DESCRIPTION
**Pull request by MonssafToukal**

### CODE REVIEW

**Comment by MonssafToukal** [comment]
@JSOBot runtests

File: test/Jenkinsfile [hunk 1]
```diff
@@ -45,22 +46,23 @@ pipeline {
 
      causeString: 'Triggered on comment',
 
-     token: "QuadraticModelsGurobi",
+     token: "QuadradicModelsGurobi",
```
Comment (dpo) [inline test/Jenkinsfile:49]: typo here

**Comment by MonssafToukal** [comment]
@JSOBot runtests

**Comment by dpo** [comment]
How come the tests passed even with the typo?

**Comment by MonssafToukal** [comment]
It seems the other pull request triggered the tests, but it shouldn't. I am trying to find why at the moment.

**Comment by MonssafToukal** [comment]
It is fixed now. Will try to execute the tests again to see if I broke anything.

**Comment by geoffroyleconte** [comment]
Thank you!

**Pull request merged**